src/App.js

```diff
@@ -60,11 +65,19 @@ export default function App() {
 
   function handleClickDetails(id) {
     setConcertId(id)
-    setActivePage('details')
+    history.push('/details')
```

Comment:
you could also use history.goBack --> it will be bring you back to the side where you've been before.